site/en/blog/private-network-access-update/index.md

```diff
@@ -1,61 +1,89 @@
 ---
 layout: 'layouts/blog-post.njk'
-title: Private Network Access (CORS-RFC1918) updates
+title: "Private Network Access update: Introducing Deprecation Trial"
 authors:
-  - agektmr
   - titouan
-description: Chrome has some updates to share about the standardization of the Private Network Access - formerly known as CORS-RFC1918 - plans to restrict private network resource accesses, and how your websites can align with those changes.
-date: 2021-03-11
-updated: 2021-06-23
+description: Chrome is deprecating access to private network endpoints from non-secure public websites in Chrome 94 as part of the Private Network Access specification. Read on for recommended actions.
```

Comment:
add:

```
tags:
- chrome-94
```

So it'll show up on the right tagged page.

Reply:
+1.